### PR TITLE
Ks tx data

### PR DESCRIPTION
**Pull request by captnseagraves**
This PR adds progress bar functionality where broken and implemented transaction history in grants/profile. It also connects all remaining data for emails.

### CODE REVIEW

File: app/grants/templates/grants/cancel.html [hunk 1]
```diff
@@ -20,7 +20,6 @@
   <head>
     {% include 'shared/head.html' %}
     {% include 'shared/cards.html' %}
-    <link rel="stylesheet" href="{% static "v2/css/bounty.css" %}">
```
Comment (thelostone-mc) [inline app/grants/templates/grants/cancel.html]: @captnseagraves this needs to added back in as the waiting loader needs css from this section.

File: app/grants/templates/grants/detail.html [hunk 2]
```diff
@@ -20,7 +20,7 @@
   <head>
     {% include 'shared/head.html' %}
     {% include 'shared/cards.html' %}
-    <link rel="stylesheet" href="{% static "v2/css/bounty.css" %}">
```
Comment (thelostone-mc) [inline app/grants/templates/grants/detail.html]: needed

File: app/grants/templates/grants/fund.html [hunk 3]
```diff
@@ -20,7 +20,6 @@
   <head>
     {% include 'shared/head.html' %}
     {% include 'shared/cards.html' %}
-    <link rel="stylesheet" href="{% static "v2/css/bounty.css" %}">
```
Comment (thelostone-mc) [inline app/grants/templates/grants/fund.html]: needed

**Comment by thelostone-mc** [comment]
@captnseagraves fixed up the email template + re-added bounty.css + fixed the progress bar conflicting css.

Here are the images for that 

screen shot 2018-12-13 at 3 41 17 pm

screen shot 2018-12-13 at 4 11 04 pm

**Comment by captnseagraves** [comment]
Thanks @thelostone-mc!

**Pull request merged**